server/kit/config.go

```diff
@@ -0,0 +1,80 @@
+package kit
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config holds info required to configure a gizmo kit.Server.
+//
+// This struct is loaded from the environment at Run and only made public to expose
+// for documentation.
+type Config struct {
+	// HealthCheckPath is used by server to init the proper HealthCheckHandler.
+	// If empty, this will default to '/healthz'.
+	HealthCheckPath string `envconfig:"GIZMO_HEALTH_CHECK_PATH"`
+
+	// MaxHeaderBytes can be used to override the default of 1<<20.
+	MaxHeaderBytes int `envconfig:"GIZMO_MAX_HEADER_BYTES"`
+
+	// ReadTimeout can be used to override the default http server timeout of 10s.
+	// The string should be formatted like a time.Duration string.
+	ReadTimeout time.Duration `envconfig:"GIZMO_READ_TIMEOUT"`
+
+	// WriteTimeout can be used to override the default http server timeout of 10s.
+	// The string should be formatted like a time.Duration string.
+	WriteTimeout time.Duration `envconfig:"GIZMO_WRITE_TIMEOUT"`
+
+	// IdleTimeout can be used to override the default http server timeout of 120s.
+	// The string should be formatted like a time.Duration string.
+	IdleTimeout time.Duration `envconfig:"GIZMO_IDLE_TIMEOUT"`
+
+	// ShutdownTimeout can be used to override the default http server shutdown timeout
+	// of 5m.
```

Comment:
Oh! I did miss the default in `loadConfig`, good catch.

5mins was just an arbitrary high setting to avoid immediate shutdown and to allow for services with long running tasks without letting them run forever.

Think we should make it less?